SportsHub/SportsHub.Api/Exceptions/CustomExceptionModels/BusinessLogicException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.Json;
+
+namespace SportsHub.Api.Exceptions.CustomExceptionModels;
+
+public class BusinessLogicException: CustomException
+{
+    public BusinessLogicException(string message)
+    {
+        Message = message;
+    }
+
+    public int StatusCode { get; } = 400; 
```

Comment:
It is not the responsibility of the exception to set the status code.

Reply:
I will set it in constructor

Reply:
I'm talking about a different thing. An exception should have information about the error, not the status code to return. You can set error code in exception middleware.